azure/services/scalesets/mock_scalesets/scalesets_mock.go

```diff
@@ -268,18 +236,18 @@ func (mr *MockScaleSetScopeMockRecorder) GetLongRunningOperationState(arg0, arg1
 }
 
 // GetVMImage mocks base method.
-func (m *MockScaleSetScope) GetVMImage() (*v1beta1.Image, error) {
+func (m *MockScaleSetScope) GetVMImage(arg0 context.Context) (*v1beta1.Image, error) {
```

Comment:
Why not `ctx`?

Reply:
This is generated code. We don't control the argument names.